src/core/ipfs/libp2p.js

```diff
@@ -31,29 +32,50 @@ module.exports = function libp2p (self) {
       self._libp2pNode.stop(callback)
     },
     swarm: {
-      peers: (callback) => {
+      peers: promisify((callback) => {
         if (!self.isOnline()) {
           return callback(OFFLINE_ERROR)
         }
 
-        callback(null, self._libp2pNode.peerBook.getAll())
-      },
+        const peers = self._libp2pNode.peerBook.getAll()
+        const mas = []
+        Object
+           .keys(peers)
+           .forEach((b58Id) => {
+             peers[b58Id].multiaddrs.forEach((ma) => {
+               // TODO this should only print the addr we are using
+               mas.push(ma)
+             })
+           })
```

Comment:
Is requiring a dependency to remove 0 lines (because the extra map to flatten step would take the exact same lines) worth it?